### DISCUSSION TITLE
NGAlert and inhibit_rules

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#37500] [Alerting]
There is any chance to include "inhibit_rules" on the routing (Notification policies)  as we have in alertmanager? 

What is the best way to create conditional triggers,m the current nested is a bit confusing if you want to inibit a group of alerts if some alert is already triggered. 

KR, 

Elton

**Comment by techwavedev** [comment]
Would be great if we could have inhibit_rules as we have in alertmanager in case we want to deactivate the routing (notification policies) of the messages in case of some pre-existent condition

like:

inhibit_rules:
  source_match:
    alertname: 'instance_down'
  target_match:
    alertname: 'some_other_triggered_alert'
  equal: [ 'namespace', 'service' ]

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
